src/Tribe/Tickets_View.php

```diff
@@ -163,8 +195,10 @@ public function update_tickets() {
 	 * @return void
 	 */
 	public function authorization_redirect() {
+		global $wp_query;
+
 		// When it's not our query we don't care
-		if ( ! tribe_is_event_query() ) {
+		if ( empty( $wp_query->tribe_is_event_query ) ) {
```

Comment:
Im not really sure... Im thinking if there is another way to go about it. The `tribe_is_modified_query` is a little bit weird for me, has too much of wide usage. Should we go with adding the filter here on Tickets? So that if it's filtered it still applies?